app/views/admin/councillor_contributions/_details.html.arb

```diff
@@ -0,0 +1,13 @@
+table do
+  tr do
+    th "Contributor Name"
+    th "Contributor Email"
+    th "Authority"
+  end
+  tr do
+    td a contributor.name, href:admin_contributor_path
+    td contributor.email
+    "#{contributor.email}"
+    td a authority.full_name
```

Comment:
It suppose to take it to authority#show page. I fixed it, but if it's not necessarily I will just make it to a text.